### DISCUSSION TITLE
" as reactions on thoses maps are largely disconnected and uninformative"

### DISCUSSION THREAD

**Discussion by Cartesian1671** [SysBioChalmers/human-maps#8] [Ideas]
Hi,

I am exploring your development and came across the statement in your subject line.  (see: Compartments maps
Correspond to a compartment in the model. Note that the Cytosol has been divided into 5 separate maps, and the Inner mitochondrial membrane compartment is included on the Mitochondria map. _**We do not provide maps for Boundary and Extracellular compartments, as reactions on thoses maps are largely disconnected and uninformative**_.

Sadly, this made no sense.
It is a claim that you made but, without checking relationships to the infectious metabolomics.  
On that note, I am a first time user and need to explore still because it is part of a project for a course.

Just thought that if I mentioned it, maybe there is a way to help-

**Comment by mihai-sysbio** [comment]
> _**We do not provide maps for Boundary and Extracellular compartments, as reactions on thoses maps are largely disconnected and uninformative**_.

Hi @Cartesian1671! Thanks for reaching out. The way these metabolic maps are constructed is independent of omics data. At the time the analysis on the map content was performed (likely around 4-5 years ago), the average connectivity of these maps was too low to be informative. Regardless, it would be great to hear about your use-case around these maps.
